app/controllers/locations_controller.rb

```diff
@@ -0,0 +1,85 @@
+class LocationsController < ApplicationController
+  before_action -> { authorize :location }
+  respond_to :html
+
+  def index
+    render_index
+  end
+
+  def show
+    render_show
+  end
+
+  def new
+    render_new
+  end
+
+  def create
+    if location.save
+      redirect_to location_path(location),
+                  notice: I18n.t('location.create.success')
```

Comment:
@LucasAnderson1984 if you use the plural `locations` then you can take advantage of some of the i18n conventions in Rails:

4.1.4 "Lazy" Lookup

Rails implements a convenient way to look up the locale inside views. When you have the following dictionary:

```
es:
  books:
    index:
      title: "Título"

you can look up the books.index.title value inside app/views/books/index.html.erb template like this (note the dot):

<%= t '.title' %>

Automatic translation scoping by partial is only available from the translate view helper method.
```